php/WP_Mock/Tools/TestCase.php

```diff
@@ -171,17 +202,97 @@ protected function mockStaticMethod( $class, $method = null ) {
 		return $expectation;
 	}
 
+	/**
+	 * @param array|object $data The post data to add to the post
+	 *
+	 * @return \WP_Post
+	 */
+	protected function mockPost( $data ) {
```

Comment:
For now this is fine, but I'd like to eventually move WP-specific mocking out of the test case. If/when we bump our version requirements to PHP 5.4 (this won't be for a while, I admit) we could move this code into a trait and any test cases that need it can just inherit the trait.

So, cool for now, but let's look towards an alternative. Maybe a `WordPressTestCase` that extends `TestCase`? Then any project test cases that need the code could extend `WordPressTestCase`?